### PR TITLE
Feature/create api to delete quiz data

### PR DESCRIPTION
**Pull request by rafaelparayno**
https://app.asana.com/0/1201258883888674/board
### Commands
_Backend_
php artisan migrate
php artisan serve

### Pre-conditions
able to see 
Starting Laravel development server: http://127.0.0.1:8000

### Expected Output
1. able to delete a quiz data
image

### CODE REVIEW

File: frontend/src/components/AdminQuizComp/QuizList.tsx [hunk 1]
```diff
@@ -1,122 +1,55 @@
...
+import Boxwidget from '@components/UI/Widget/Boxwidget/Boxwidget';
+import Quiz from '../../models/quizModels';
+import Spinner from '@components/UI/Spinner/Spinner';
+import { ThunkDispatch } from 'redux-thunk';
+import { Action } from '../../store/actions/index'
+import { bindActionCreators } from 'redux';
+import { editQuizAdminModal } from '../../store/actions/action-creator/'
+import { connect } from 'react-redux'
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizList.tsx:14]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizList.tsx [hunk 2]
```diff
@@ -1,122 +1,55 @@
...
+interface RProps {
+  loading: boolean;
+  data: Quiz[] | null;
+}
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizList.tsx:20]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizList.tsx [hunk 3]
```diff
@@ -1,122 +1,55 @@
...
+type Props = RProps & LinkDispatchProps;
+
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizList.tsx:24]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizList.tsx [hunk 4]
```diff
@@ -1,122 +1,55 @@
...
+const QuizList: FC<Props> = ({ loading, data, editQuizAdminModal }) => {
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizList.tsx:27]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizModal.tsx [hunk 5]
```diff
@@ -1,125 +1,135 @@
...
+} from "@heroicons/react/solid";
+import { RootState } from '../../store/reducers'
+import { ThunkDispatch } from 'redux-thunk';
+import { Action } from '../../store/actions/index'
+import { bindActionCreators } from 'redux';
+import { editQuizAdminModal, saveQuizData, updateQuizData } from '../../store/actions/action-creator/'
+import { connect } from 'react-redux'
+import Quiz from '../../models/quizModels';
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizModal.tsx:14]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizModal.tsx [hunk 6]
```diff
@@ -1,125 +1,135 @@
...
+interface RProps {
+
+}
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizModal.tsx:17]: if not using delete this 
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizModal.tsx [hunk 7]
```diff
@@ -1,125 +1,135 @@
...
+
+type Props = RProps & LinkDispatchProps & LinkStateProps;
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizModal.tsx:21]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizModal.tsx [hunk 8]
```diff
@@ -1,125 +1,135 @@
...
+                </div>
+                <div className="flex items-center justify-between">
+                    <button disabled={SaveLoading} onClick={() => saveClick()} className="bg-gray-600 text-lg hover:bg-gray-800 text-white font-bold py-2 px-10 rounded focus:outline-none focus:shadow-outline" type="button">
+                        Save{" "}  {SaveLoading ? (
+
+                            <svg className="animate-spin -ml-1 mr-3 h-5 w-5 text-white" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
+                                <circle className="opacity-25" cx="12" cy="12" r="10" stroke="currentColor" stroke-width="4"></circle>
+                                <path className="opacity-75" fill="currentColor" d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"></path>
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizModal.tsx:98]: create svg for this

File: frontend/src/components/AdminQuizComp/QuizModal.tsx [hunk 9]
```diff
@@ -1,125 +1,135 @@
...
+        formData.set('title', input.title);
+        formData.set('description', input.description);
+
+        input.id ?
+            updateQuizData(input, input.id && input.id)
+            : saveQuizData(formData);
+
+    }
+
+
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizModal.tsx:66]: remove extra space
```suggestion
```

File: frontend/src/components/AdminQuizComp/QuizModal.tsx [hunk 10]
```diff
@@ -1,125 +1,135 @@
...
+        setInput({
+            ...input,
+            [name]: value
+        })
+    }
+
+    const saveClick = () => {
+        if (input && !input.title || input && !input.description) {
+            return
```
Comment (ejanton-sun) [inline frontend/src/components/AdminQuizComp/QuizModal.tsx:52]:
```suggestion
            return;
```

**Review by ejanton-sun — CHANGES_REQUESTED** [review]

File: frontend/src/components/AdminQuizComp/QuizModal.tsx (hunk 8 above)
Comment (rafaelparayno) [reply frontend/src/components/AdminQuizComp/QuizModal.tsx:98]: already done this sir on pr 10

**Review by ejanton-sun — APPROVED** [review]

**Pull request merged**